src/pytorch/neighbors/getNeighborPairs.py

```diff
@@ -63,18 +80,27 @@ def getNeighborPairs(positions: Tensor, cutoff: float, max_num_neighbors: int =
         If an atom pair is separated by a larger distance than the cutoff,
         the distance is set to `NaN`.
 
+    number_found_pairs: `Optional[torch.Tensor]`
+        Present  if check_errors=True.  Contains the  total number  of
```

Comment:
For consistency, the number of pairs should be reported always.

Reply:
This changes the interface beyond the new functionality, invalidating any code that uses this function up until that moment.
Every test fails with:
```bash
FAILED TestNeighbors.py::test_is_cuda_graph_compatible - ValueError: too many values to unpack (expected 3)
```